buildSrc/src/main/java/Dependencies.kt

```diff
@@ -60,4 +60,19 @@ object Samples {
         const val compiler = "androidx.room:room-compiler:${version}"
         const val rxjava = "androidx.room:room-rxjava2:${version}"
     }
+
+    object OkHttp {
+        private const val version = "4.9.3"
+        const val okhttp = "com.squareup.okhttp3:okhttp:${version}"
+    }
+
+    object Timber {
+        private const val version = "5.0.1"
```

Comment:
Our lib currently uses `4.7.1`

Reply:
not sure, if our CI compiles with a different version it means we cannot guarantee that it works with the transitive dependency, maybe a matrix then?

Reply:
hmm, I think I'm missing the point. The transitive dependency works, because this is how it is in the sentry-java repo now, right?

Reply:
I actually added a test that uses different versions of okhttp/timber/fragment. I think that should be enough? https://github.com/getsentry/sentry-android-gradle-plugin/blob/feat/add-dependencies/plugin-build/src/test/kotlin/io/sentry/android/gradle/SentryPluginAutoInstallTest.kt